templates/CMakeLists.txt.template

```diff
@@ -441,6 +441,9 @@
   % if lib.language == 'c++':
     PRIVATE <%text>${_gRPC_PROTO_GENS_DIR}</%text>
   % endif
+  % if lib.name in ['upb']:
```

Comment:
I didn't see a entry in build.yaml for specifying include paths. Is this the right idiom for this?